libcxx/src/include/overridable_function.h

```diff
@@ -13,6 +13,10 @@
 #include <__config>
 #include <cstdint>
 
+#if defined(__arm64e__) && __has_feature(ptrauth_calls)
```

Comment:
It's probably easier to define a single macro for this than to repeat it everywhere.

Reply:
Also we should drop the `defined(__arm64e__)` check to allow other platforms, same as arm64